qtpy/tests/test_qtwidgets.py

```diff
@@ -130,3 +135,74 @@ def test_opengl_imports():
     QtPy makes QOpenGLWidget available in QtWidgets to maintain compatibility.
     """
     assert QtWidgets.QOpenGLWidget is not None
+
+@pytest.mark.skipif(sys.platform == 'darwin' and sys.version_info[:2] == (3, 7) and (PYQT5 or PYSIDE2), reason="test crashes on macOS CI with Python 3.7 with 'Illegal instruction: 4'")
```

Comment:
```suggestion

@pytest.mark.skipif(
    sys.platform == 'darwin' and sys.version_info[:2] == (3, 7) and
    (PYQT5 or PYSIDE2),
    reason="Crashes on macOS with Python 3.7 with 'Illegal instruction: 4'")
```

This line was super long. We prefer lines to less than 80 columns wide.